pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/AbstractIgnoredAnnotRule.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import net.sourceforge.pmd.lang.java.ast.Annotatable;
+import net.sourceforge.pmd.properties.StringMultiProperty;
+
+public abstract class AbstractIgnoredAnnotRule extends AbstractJavaRule {
```

Comment:
I'd avoid abbreviations and just name the class `AbstractIgnoredAnnotationsRule`.